src/index.js

```diff
@@ -1,36 +1,76 @@
-const tasks = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Completed TO Do list project',
-    completed: true,
-    index: 2,
-  },
-];
+import './style.css';
 
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+const completeButton = document.querySelector('.complete');
+
+// add task
 document.addEventListener('DOMContentLoaded', () => {
-  const todoList = document.getElementById('todo-list');
-  todoList.innerHTML = '';
-  todoList.style.listStyle = 'none';
+  const removeTask = (event) => {
+    const li = event.target.parentNode;
+    todoList.removeChild(li);
+  };
 
-  tasks.sort((a, b) => a.index - b.index);
+  const editTask = (event) => {
+    const li = event.target.parentNode;
+    const taskValue = li.firstChild.textContent.trim();
+    const newTaskValue = prompt('Edit task:', taskValue);
+    if (newTaskValue !== null && newTaskValue.trim() !== '') {
+      li.firstChild.textContent = newTaskValue.trim();
+    } else {
+      return;
+    }
 
-  tasks.forEach((task) => {
-    const li = document.createElement('li');
-    li.style.border = '1px solid #ddd';
-    li.style.display = 'block';
-    li.style.padding = '10px 0px 10px 50px';
-    li.style.marginLeft = '-40px';
-    li.style.marginRight = '50vw';
+    const popupDiv = document.createElement('div');
+    popupDiv.classList.add('popup');
+    const heading = document.createElement('h2');
+    heading.textContent = 'Edit task';
+    const input = document.createElement('input');
+    input.type = 'text';
+    input.value = newTaskValue.trim();
+    const saveButton = document.createElement('button');
+    saveButton.textContent = 'Save';
+    saveButton.addEventListener('click', () => {
+      const editedTaskValue = input.value.trim();
+      if (editedTaskValue !== '') {
+        li.firstChild.textContent = editedTaskValue;
+        popupDiv.remove();
+      }
+    });
+    popupDiv.appendChild(heading);
+    popupDiv.appendChild(input);
+    popupDiv.appendChild(saveButton);
+    document.body.appendChild(popupDiv);
+  };
 
+  const addTask = () => {
+    const taskValue = taskInput.value.trim();
+
+    const li = document.createElement('li');
```

Comment:
- [ ] Well done :clap: you did a very nice job :+1: but there are some issues in your project kindly fix the following issues :slightly_smiling_face: 

- All changes to the To-Do List should be saved in `local storage` so kindly when you add new data it should be stored in the `local storage`.
- After storing your data `local storage` then when the user removes the data then Deleting a task should update all remaining items' `indexes`, so they represent the current list order and are unique(i.e. if you're deleting the `first task` index `1` from the `list`, the index of the `next task(2)` should set to `1`).
- By default, new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Please add an array of objects whose values should be `completed`, `description` and `index` and store these values in `local storage`